app/partials/_navbar.html.haml

```diff
@@ -0,0 +1,22 @@
+.container-fluid
+  .row
+    .col-md-12
+      %nav.navbar.navbar-default{:role => "navigation"}
+        .navbar-header
+          %button.navbar-toggle{"data-target" => "#bs-example-navbar-collapse-1", "data-toggle" => "collapse", :type => "button"}
+            %span.sr-only Toggle navigation
+            %span.icon-bar
+            %span.icon-bar
+            %span.icon-bar
+          %a.navbar-brand Best Slow Food
+        #bs-example-navbar-collapse-1.collapse.navbar-collapse
+          %ul.nav.navbar-nav
+            %li.active
+              = link_to "Home", root_path
+              = link_to "Create dish", new_dish_path
+              = link_to 'Show cart', carts_path
```

Comment:
Thank you for using single quotes. Now, please change the other links. Apart from other reasons, consistency is a virtue.